exercises/practice/tournament/.docs/instructions.append.md

```diff
@@ -0,0 +1,131 @@
+# placeholder
+
+## Sorting in AWK
+
+This exercise requires the output to be sorted in a particular way.  Like
```

Comment:
The Python track uses one sentence per line in markdown files instead of using fixed width. Not sure how you feel about one over the other.

Reply:
I've [read about that](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line) and I see the merits behind it. But I think that markdown files have to work equally being read as a plain text file, and the jagged line lengths are quite distracting I find.